src/internal/operations/backup.go

```diff
@@ -589,6 +596,11 @@ func (op *BackupOperation) persistResults(
 			opStats.writeErr)
 	}
 
+	if opStats.gc == nil {
```

Comment:
We check if either are nil on line ~582
```go
if opStats.readErr != nil || opStats.writeErr != nil {
		op.Status = Failed
```
Should we only be checking if gc.successful at this point... after the guard check of course?

Reply:
No need at this time.